tests/modules/nf-core/arcashla/extract/test.yml

```diff
@@ -0,0 +1,22 @@
+- name: arcashla extract test_arcashla_extract
+  command: nextflow run ./tests/modules/nf-core/arcashla/extract -entry test_arcashla_extract -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/arcashla/extract/nextflow.config
+  tags:
+    - arcashla
+    - arcashla/extract
+  files:
+    - path: output/arcashla/test.paired_end.sorted.extracted.1.fq.gz
+    - path: output/arcashla/test.paired_end.sorted.extracted.2.fq.gz
+    - path: output/arcashla/test.log
```

Comment:
Any string you could include indicating completeness? (e.g. `Finished`) if a stable one exists?

Reply:
Not really but I will add the header of the last processing step.